ui/components/app/toast-master/selectors.ts

```diff
@@ -62,25 +64,40 @@ export function selectShowPrivacyPolicyToast(state: State): {
     newPrivacyPolicyToastShownDate,
     onboardingDate,
   } = state.metamask || {};
-  const newPrivacyPolicyDate = new Date(PRIVACY_POLICY_DATE);
-  const currentDate = new Date(Date.now());
+
+  const remoteFeatureFlags = state.metamask?.remoteFeatureFlags || {};
```

Comment:
You can use the `getRemoteFeatureFlags` selector from `ui/selectors/remote-feature-flags.ts`

Reply:
I will revisit this. TY